crates/bitwarden-json/src/client.rs

```diff
@@ -71,8 +75,11 @@ impl Client {
 
     fn parse_settings(settings_input: Option<String>) -> Option<ClientSettings> {
         if let Some(input) = settings_input.as_ref() {
-            if let Ok(settings) = serde_json::from_str(input) {
-                return Some(settings);
+            match serde_json::from_str(input) {
+                Ok(settings) => return Some(settings),
+                Err(e) => {
+                    eprintln!("Failed to parse settings: {}", e);
```

Comment:
Do we know what happens with eprint in wasm or other environments?

Reply:
It should work fine in everything but wasm, which doesn't have a standard output by default. We can add the log crate to bitwarden-json and use that, which should work.

Reply:
Changed it to log::error, which works when used with `console_log` in the wasm crate.